### ISSUE TITLE
Multi language file fields

### ISSUE THREAD

**Issue by Jehfen** [processwire/processwire-requests#372]
### Short description of the enhancement

Several common fields in PW are multi language, except for the File (and image) Field.

### Current vs. suggested behavior

I know you can make language alternate fields to accomplish this behavior, but frankly that's a hassle.
Using language tabs as a default would make it an automatic task without any fuss.

### Why would the enhancement be useful to users?

It would be much more consistent with language behavior of other common fields, and it would be more consistent with the defined website languages in general.

Apart from the burden to create several language alternate fields, it would take a lot less space in the editor and would confuse the user less.
Multi language pdf downloads are very common, but also images would benefit from upload in language tabs; some images could contain localized text, or you want to show all images in the english version, but only a subset in another language.

**Comment by jmartsch** [comment]
Yes. This would be great. I have a multilingual site, where my client wanted different images for a product according to the language, and it was very cumbersome to come up with a solution.
And now the input in the admin panel is clunky, because I have multiple fields for images.

**Comment by romaincazier** [comment]
I just faced this situation and went for the solution described here, using tags : https://processwire.com/blog/posts/processwire-updates-and-new-field-types/

**Comment by Jehfen** [comment]
> I just faced this situation and went for the solution described here, using tags : https://processwire.com/blog/posts/processwire-updates-and-new-field-types/

Interesting solution, in the explanation there was:
_...."For instance, we could support an option for predefined tags,"_
If that's implemented it could pass for a multilanguage file field, otherwise users will definitely type the wrong tags.

**Comment by romaincazier** [comment]
> otherwise users will definitely type the wrong tags

Actually if you go for predefined tags (which is supported now, I don't since when), there's no such risk. See these screenshots from the options and the actual field (sorry for the english / french mix) : 

![Capture d’écran 2020-11-05 à 10 11 52](https://user-images.githubusercontent.com/6616448/98221196-a2415480-1f4f-11eb-84c1-03e33cdd0ce9.jpg)
![Capture d’écran 2020-11-05 à 10 12 17](https://user-images.githubusercontent.com/6616448/98221202-a3728180-1f4f-11eb-8f11-c19e734fa932.jpg)

**Comment by Jehfen** [comment]
Oh wow, didn't know that those predefined tags were already there. Perfect, thanks for the tip.

**Comment by Jehfen** [comment]
Maybe I'll just leave it open, since it's still a feature request to have multi-language file fields, apart from the predefined image tags.

**Comment by tobaco** [comment]
You could also use language alternate fields for this. 
see: https://processwire.com/docs/multi-language-support/multi-language-fields/#language-alternate-fields
